src/pages/validator/external-chains/blast.mdx

```diff
@@ -0,0 +1,206 @@
+# Blast
+
+Instructions to set up your blast node.
+
+## Requirements
+
+- [Setup your Axelar validator](/validator/setup)
+- Minimum hardware requirements: 8 AWS vCPU+, 8GB RAM, 512GB of SSD free storage space.
+- MacOS or Ubuntu 22.04 (tested on 22.04)
+- [Official Documentation](https://github.com/blast-io/deployment)
+- [Official Documentation](https://docs.blast.io/tools/node-providers)
```

Comment:
"Official Documentation" -> "Official Documentation for Node Providers"

Reply:
Renamed